Makefile

```diff
@@ -3,7 +3,7 @@ SDIR=src
 TDIR=pytests
 # compilers
 TSC=./node_modules/typescript/bin/tsc
-TSCFLAGS=-t ES3 --sourceMap --module commonjs --noImplicitAny
```

Comment:
Oh btw, the makefile errors I was talking about result from this change that I made.
https://github.com/Mdrbhatti/Ninia/blob/master/Makefile#L62